factfinder/geography/2010_to_2020.py

```diff
@@ -62,12 +66,12 @@ def agg_moe(x):
     def convert_moe(e_2010, m_2010, e_2020, ratio):
         if ratio == 1:
             return m_2010
-        elif e_2020 == 0:
+        elif e_2010 == 0:
```

Comment:
This should be the 2020 estimate.

Reply:
ohhh the middle one is correct, I see, i thought it was the bottom one